documents/Libraries/stdlib/osl/mx_power_color.osl

```diff
@@ -1,12 +1,12 @@
 // Open Shading Language : Copyright (c) 2009-2017 Sony Pictures Imageworks Inc., et al.
 // https://github.com/imageworks/OpenShadingLanguage/blob/master/LICENSE
-// 
-// MaterialX specification (c) 2017 Lucasfilm Ltd. 
+//
+// MaterialX specification (c) 2017 Lucasfilm Ltd.
 // http://www.materialx.org/
 
 #include "mx_funcs.h"
 
-shader mx_exponent_color
+shader mx_power_color
```

Comment:
There are changes coming soon from Sony to take care of all updates on the reference OSL side. Did you need this change to fix some test failure? Otherwise we could revert the changes in reference OSL to avoid conflicts with their changes.

Reply:
This change probably is from ILM and just part of the merge from master.